evm/src/NttManager/NttManager.sol

```diff
@@ -30,6 +30,7 @@ import {ManagerBase} from "./ManagerBase.sol";
 ///  - the amount
 ///  - the recipient chain
 ///  - the recipient address
+///  - the refund address: the address to refund the user for "dummy" attestations
```

Comment:
What is a "dummy" attestation?

Reply:
Also, folks will be refunded for attestations that succeed, but the gas limit is configured to be too high. It would be nice to mention that.